source/MaterialXGenGlsl/GlslShaderGenerator.cpp

```diff
@@ -457,14 +456,15 @@ void GlslShaderGenerator::emitPixelStage(const ShaderGraph& graph, GenContext& c
         emitLineBreak(stage);
     }
 
-    // Emit uv transform function
-    if (!context.getOptions().fileTextureVerticalFlip)
+    // Set the include file to use for uv transformations,
+    // depending on the vertical flip flag.
+    if (context.getOptions().fileTextureVerticalFlip)
     {
-        emitInclude("stdlib/" + GlslShaderGenerator::LANGUAGE + "/lib/mx_transform_uv.glsl", context, stage);
```

Comment:
This would always emit the mx_transform_uv function, even it's not used. Now each implementation that needs it is doing the #include in its source file instead.